packages/web-sockets/config/environment.js

```diff
@@ -0,0 +1,16 @@
+// External dependencies
```

Comment:
I think this should be addressed in the shared package, maybe we can create a PR later to do it.